src/data/complete.queryMap.json

```diff
@@ -1057,22 +1057,5 @@
-  "81b5d3db0cdf39fdb4dde2257615ae3f": "query FairArtworksQuery(\n  $fairID: String!\n) {\n  fair(id: $fairID) {\n    ...FairArtworks_fair\n    id\n  }\n}\n\nfragment FairArtworks_fair on Fair {\n  id\n  internalID\n  slug\n  ...FilteredInfiniteScrollGrid_entity\n}\n\nfragment FilteredInfiniteScrollGrid_entity on EntityWithFilterArtworksConnectionInterface {\n  id\n  filterArtworksConnection(first: 10, medium: \"*\", priceRange: \"*-*\", aggregations: [MEDIUM, PRICE_RANGE, TOTAL]) {\n    edges {\n      node {\n        id\n        __typename\n      }\n      cursor\n    }\n    ...Filters_filteredArtworks\n    ...InfiniteScrollArtworksGrid_connection\n    pageInfo {\n      endCursor\n      hasNextPage\n    }\n    id\n  }\n}\n\nfragment Filters_filteredArtworks on FilterArtworksConnection {\n  aggregations {\n    slice\n    counts {\n      name\n      value\n    }\n  }\n}\n\nfragment InfiniteScrollArtworksGrid_connection on ArtworkConnectionInterface {\n  pageInfo {\n    hasNextPage\n    startCursor\n    endCursor\n  }\n  edges {\n    __typename\n    node {\n      slug\n      id\n      image {\n        aspectRatio\n      }\n      ...ArtworkGridItem_artwork\n    }\n  }\n}\n\nfragment ArtworkGridItem_artwork on Artwork {\n  title\n  date\n  sale_message: saleMessage\n  is_in_auction: isInAuction\n  is_biddable: isBiddable\n  is_acquireable: isAcquireable\n  is_offerable: isOfferable\n  slug\n  sale {\n    is_auction: isAuction\n    is_live_open: isLiveOpen\n    is_open: isOpen\n    is_closed: isClosed\n    display_timely_at: displayTimelyAt\n    id\n  }\n  sale_artwork: saleArtwork {\n    current_bid: currentBid {\n      display\n    }\n    id\n  }\n  image {\n    url(version: \"large\")\n    aspect_ratio: aspectRatio\n  }\n  artists(shallow: true) {\n    name\n    id\n  }\n  partner {\n    name\n    id\n  }\n  href\n}\n"
+  "d13130a7081378f93f0668ff3fa60453": "query ShowsQuery(\n  $count: Int!\n  $cursor: String\n) {\n  me {\n    ...Shows_me_1G22uz\n    id\n  }\n}\n\nfragment Shows_me_1G22uz on Me {\n  followsAndSaves {\n    shows: showsConnection(first: $count, after: $cursor) {\n      pageInfo {\n        startCursor\n        endCursor\n        hasPreviousPage\n        hasNextPage\n      }\n      edges {\n        node {\n          id\n          ...ShowItemRow_show\n          __typename\n        }\n        cursor\n      }\n    }\n  }\n}\n\nfragment ShowItemRow_show on Show {\n  id\n  slug\n  internalID\n  is_followed: isFollowed\n  name\n  isStubShow\n  partner {\n    __typename\n    ... on Partner {\n      name\n      profile {\n        image {\n          url(version: \"square\")\n        }\n        id\n      }\n    }\n    ... on Node {\n      id\n    }\n    ... on ExternalPartner {\n      id\n    }\n  }\n  href\n  exhibition_period: exhibitionPeriod\n  status\n  cover_image: coverImage {\n    url\n    aspect_ratio: aspectRatio\n  }\n  is_fair_booth: isFairBooth\n  start_at: startAt\n  end_at: endAt\n}\n"
```

Comment:
Are these changes intentional?

Reply:
No. I think they are auto-generated. Should they be removed?